src/execute-js-func.js

```diff
@@ -24,8 +25,10 @@ const BrowserWindow = isBrowser ?
   require('electron').remote.BrowserWindow;
 
 let nextId = 1;
+const hashIds = new Hashids();
+
 function getNextId() {
-  return (process.pid << 32) | (nextId++);
+  return hashIds.encode(process.pid, nextId++);
```

Comment:
`process.pid + "-" + nextId++` would suffice, wouldn't it? might be nice to drop the dependency?